CONTRIBUTING.md

```diff
@@ -1,65 +1,98 @@
 # Contribution
 
 ## Table of Contents
+* [Labels](#labels)
 * [Issues & Labels](#issues)
 * [Pull Requests](#prs)
-* [Approval Process](#approvals)
+* [Pull Requests Approval Process](#approvals)
+* [Idle Pull Requests Policy](#idle)
+* [Unresolved Pull Requests Policy](#unresolved)
+
+<a name="labels"></a>
+### Labels
+Following labels should be correctly used for all issues and PRs to indicate the status of them and to assist automation pipelines to correctly apply CNTT processes.
+
+  - **Roadmap Item**: This label applies to both issues and PRs to indicate that the issue/PR is for a roadmap item for an upcoming release.
+  - **Fix needed**: This label applies to both issues and PRs to indicate that the issue/PR is to fix an existing bug.
+  - **Backlog**: This label applies to issues only to indicate that the issue is to be added to a backlog of features for future releases (to be considered during technical steering meeting).
+  - **Workstream Name**: This label applies to both issues and PRs to indicate which project the issue/PR is targeting.
+  - **Chapter Number**: This label applies to PRs (and optionally to issues) to indicate which chapter the PR is targeting.
+  - **General**: This label applies to both issues and PRs to indicate that this issue/PR is not corresponding to any particular chapter/file.
+  - **Major/Minor**: This label applies to both issues and PRs to indicate if the issue/PR is addressing a major or minor change. 
+  - **WIP**: This label applies to PRs to indicate that this PR is still being developed and no reviews/approvals are expected.
+  - **Enhancement**: This label applies to both issues and PRs to indicate that the issue/Pr is proposing an Enhancement.
+  - **Returned**: This label applies to both issues and PRs to indicate that the issue/PR has been sent back to authoer (e.g. more information needed, not a high priority, no longer valid, etc).
+  - **Idle**: This label will be **automatically** applied (by a GitHub work flow or by TSC Leads) to both issues/PRs to indicate that no activity has been observed on the issue/PR for some determined time frame. This an trigger an action to close the issue/PR.
```

Comment:
Typo: "This an trigger an action to close the issue/PR." ---> "This triggers an action to close the issue/PR."

Reply:
fixed